### PR TITLE
docs(SRV): add note on external container registries MTA-5165

### PR DESCRIPTION
**Pull request by SamyOubouaziz**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: faq/containerregistry.mdx [hunk 1]
```diff
@@ -16,6 +16,20 @@ Scaleway Container Registry is a fully managed mutualized Container Registry, de
...
+
+Scaleway Container Registry is billed based on stored images size, and outgoing data transfer.
+
+|                | Stored Images   | Outgoing data transfer                           | Incoming data transfert |
+|----------------|-----------------|--------------------------------------------------|-------------------------|
+| Private images | €0.027/GB/month | Inter-regional: €0.033/GB - Intra-regional: free | Free                    |
+| Public images  | Free up to 75GB | Inter-regional: free - Intra-regional: free      | Free                    |
+
+<Message type="note">
+- Inter-regional traffic: AMS ↔ PAR
```
Comment (thomas-tacquet) [inline faq/containerregistry.mdx:29]: add that it's an example a not limited to.

File: serverless/containers/troubleshooting/common-errors.mdx [hunk 2]
```diff
@@ -56,3 +56,13 @@ The new deploy failed, and the [fallback mechanism has been triggered](/serverle
...
 
 Identify the element that caused the deployment to fail, fix the error, and deploy the container again.
+
+## Issues when retrieving an external image
+
+### Cause
+
+Serverless products support external public registries (such as [Docker Hub](https://hub.docker.com/)), but we do not recommend using them due to uncontrolled rate limiting, which can lead to failures when starting resources, unexpected usage conditions, and pricing changes.
+
+### solution
```
Comment (thomas-tacquet) [inline serverless/containers/troubleshooting/common-errors.mdx:66]:
```suggestion
### Solution
```

File: serverless/jobs/how-to/create-job-from-external-registry.mdx [hunk 3]
```diff
@@ -14,10 +14,12 @@ categories:
   - jobs
 ---
 
-Scaleway allows you to create jobs from external public [container registries](/containers/container-registry/concepts/#registry), such as Docker Hub, AWS container registries, GitLab container registry, etc.
+Scaleway Serverless Jobs allows you to create jobs from external public [container registries](/containers/container-registry/concepts/#registry), such as Docker Hub, AWS container registries, GitLab container registry, etc.
 
-<Message type="note">
-  Private container registries are currently not supported.
+Private container registries are currently not supported.
```
Comment (thomas-tacquet) [inline serverless/jobs/how-to/create-job-from-external-registry.mdx:19]:
```suggestion
Private external container registries are currently not supported.
```

File: serverless/jobs/troubleshooting/common-errors.mdx [hunk 4]
```diff
@@ -20,3 +20,13 @@ categories:
...
 
 - Make sure your deployment does not exceed the limitations of [Serverless Jobs](/serverless/jobs/reference-content/jobs-limitations/).
+
+## Issues when retrieving an external image
+
+### Cause
+
+Serverless products support external public registries (such as [Docker Hub](https://hub.docker.com/)), but we do not recommend using them due to uncontrolled rate limiting, which can lead to failures when starting resources, unexpected usage conditions, and pricing changes.
+
+### solution
```
Comment (thomas-tacquet) [inline serverless/jobs/troubleshooting/common-errors.mdx:30]:
```suggestion
### Solution
```

**Review by thomas-tacquet — APPROVED** [review]

File: faq/containerregistry.mdx (hunk 1 above)
Comment (SamyOubouaziz) [inline faq/containerregistry.mdx:29]:
```suggestion
- Inter-regional traffic: AMS ↔ PAR, WAW ↔ PAR, AMS ↔ WAW
```

File: faq/containerregistry.mdx [hunk 5]
```diff
@@ -16,6 +16,20 @@ Scaleway Container Registry is a fully managed mutualized Container Registry, de
...
+Scaleway Container Registry is billed based on stored images size, and outgoing data transfer.
+
+|                | Stored Images   | Outgoing data transfer                           | Incoming data transfert |
+|----------------|-----------------|--------------------------------------------------|-------------------------|
+| Private images | €0.027/GB/month | Inter-regional: €0.033/GB - Intra-regional: free | Free                    |
+| Public images  | Free up to 75GB | Inter-regional: free - Intra-regional: free      | Free                    |
+
+<Message type="note">
+- Inter-regional traffic: AMS ↔ PAR, WAW ↔ PAR, or AMS ↔ WAW
+- Intra-regional traffic: PAR ↔ PAR, or AMS ↔ AMS
```
Comment (SamyOubouaziz) [inline faq/containerregistry.mdx:30]:
```suggestion
- Intra-regional traffic: PAR ↔ PAR, WAW ↔ WAW or AMS ↔ AMS
```

File: faq/containerregistry.mdx [hunk 6]
```diff
@@ -16,6 +16,20 @@ Scaleway Container Registry is a fully managed mutualized Container Registry, de
...
+Scaleway Container Registry is billed based on stored images size, and outgoing data transfer.
+
+|                | Stored Images   | Outgoing data transfer                           | Incoming data transfert |
+|----------------|-----------------|--------------------------------------------------|-------------------------|
+| Private images | €0.027/GB/month | Inter-regional: €0.033/GB - Intra-regional: free | Free                    |
+| Public images  | Free up to 75GB | Inter-regional: free - Intra-regional: free      | Free                    |
+
+<Message type="note">
+- Inter-regional traffic: AMS ↔ PAR, WAW ↔ PAR, or AMS ↔ WAW
+- Intra-regional traffic: PAR ↔ PAR, WAW ↔ WAW or AMS ↔ AMS
```
Comment (SamyOubouaziz) [inline faq/containerregistry.mdx:30]:
```suggestion
- Intra-regional traffic: PAR ↔ PAR, WAW ↔ WAW, or AMS ↔ AMS
```

**Pull request merged**